EcpBtBot/DialogFlow/Discussion_names.py

```diff
@@ -0,0 +1,5 @@
+ 
+class Discussion_names:     
```

Comment:
it doesn't seem lile you need this class. no?

Reply:
we will check this later, we want to make our code more interactive by keeping the list of names and client_tokens in an external file (.csv for exemple)